Applications/Spire/Source/Ui/DecimalBox.cpp

```diff
@@ -529,6 +549,35 @@ void DecimalBox::keyPressEvent(QKeyEvent* event) {
   QWidget::keyPressEvent(event);
 }
 
+void DecimalBox::mousePressEvent(QMouseEvent* event) {
+  if(m_data && event->button() == Qt::LeftButton) {
+    event->accept();
+    if(m_data->m_up_button->geometry().contains(event->pos())) {
+      m_tick = TickIndicator::UP;
```

Comment:
Can you explain why ```m_tick``` is set to ```TickIndicator::UP```? ```m_tick``` is intended strictly for style selection.

Reply:
I initially intended to use it as flags to indicate which button was being held during the timerEvent, but it turned out to be unnecessary. I'll remove it.